index.bs

```diff
@@ -691,7 +691,7 @@ An event-level trigger configuration is a [=struct=] with the following items:
 <h3 id="aggregation-coordinator-header">Aggregation coordinator</h3>
 
 An <dfn>aggregation coordinator</dfn> is one of a user-agent-determined [=set=]
```

Comment:
Does this still need to be described as a set in the specification, or does it suffice to say that an aggregation coordinator is an origin?

Reply:
I think it's still useful to describe it as a set as only an origin in the allowlist will be valid?